src/script/lua_api/l_noise.cpp

```diff
@@ -499,6 +511,49 @@ int LuaPcgRandom::l_rand_normal_dist(lua_State *L)
 	return 1;
 }
 
+int LuaPcgRandom::l_get_state(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+
+	LuaPcgRandom *o = checkObject<LuaPcgRandom>(L, 1);
+
+	u64 state[2];
+	o->m_rnd.getState(state);
+
+	std::stringstream s_state;
+	
+	s_state << std::hex << std::setw(64) << std::setfill('0') << state[0];
```

Comment:
Why a width of 64? 64 bits can be encoded in 16 nibbles (hex characters) of 4 bits each.

Reply:
good point, fixed.